judges/quality-of-service/quality-of-service.rb

```diff
@@ -105,3 +105,21 @@ def lifetime(type)
   end
 end
 f.average_backlog_size = issues.empty? ? 0 : issues.inject(&:+) / issues.size
+
+# Rejection PR rate
+all_prs_count = 0
+unmerged_prs_count = 0
+Fbe.unmask_repos.each do |repo|
+  all_q = "repo:#{repo} type:pr closed:>#{$SINCE.utc.iso8601[0..10]}"
+  all_prs_count += Fbe.octo.search_issues(all_q)[:total_count]
+
```

Comment:
@Yegorov it's not a good idea to use empty lines inside methods: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html